src/components/KonnectorSync.jsx

```diff
@@ -11,20 +11,53 @@ function getDateLabel({ date, t, f }) {
   return f(DateFns.parse(date), t('account.message.synced.date_format'))
 }
 
+const months = [
+  'Jan',
+  'Feb',
+  'Mar',
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec'
+]
+const dows = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
+
+const humanCron = cronSpec => {
+  try {
+    const [second, minute, hour, day, month, dayOfWeek] = cronSpec.split(' ')
+    const timePart =
+      hour !== '*' && minute !== '*'
+        ? `at ${hour}:${minute}`
+        : hour !== '*' ? `${hour}h` : `Any hour on ${minute}minute`
+    const monthPart = month !== '*' ? `during ${months[month]}` : ''
+    const dowPart = dayOfWeek !== '*' ? `on ${dows[dayOfWeek]}` : ''
+    return [timePart, monthPart, dowPart].join(' ')
+  } catch (e) {
+    return JSON.stringify(e)
+  }
+}
+
 export const KonnectorSync = ({
   t,
   f,
   frequency,
   lastSuccessDate,
   maintenance,
   submitting,
-  onForceConnection
+  onForceConnection,
+  trigger
 }) => {
   const lastSyncMessage =
     (submitting && t('account.message.synced.syncing')) ||
     (!lastSuccessDate && t('account.message.synced.unknown')) ||
     (lastSuccessDate && getDateLabel({ date: lastSuccessDate, t, f })) ||
     null
+  console.log('Scheduled execution :', humanCron(trigger.arguments))
```

Comment:
Do we have to log the schedule here? In production we should avoid logging things which are not warnings or errors.